honeybee_radiance_command/gensky.py

```diff
@@ -0,0 +1,228 @@
+"""gensky command."""
+
+from .options.gensky import GenskyOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+
+
+class Gensky(Command):
+    """gensky command."""
+
+    __slots__ = ('_month', '_day', '_time', '_time_zone', '_solar_time', '_input')
+
+    def __init__(self, month, day, time, time_zone=None, solar_time=False,
+                 options=None, output=None):
+        """Command.
+
+        Args:
+            month: An integer representing the number of the month. Count starts from 01.
+            day: An integer representing the number of the day in a month. Count starts
+                from 01.
+            time: A float representing hour and minute.
+            time_zone: A three letter text representing the time zone.
+                Following are acceptable time zones with their corresponding hour
+                differences from Greenwhich mean time;
+                YST +9, PST +8, MST +7, CST +6, EST +5, GMT 0, CET -1, EET -2, AST -3,
+                GST -4, IST -5.5, JST -9, NZST -12, YDT +8, PDT +7, MDT +6, CDT +5,
+                EDT +4, BST -1, CEST -2, EEST -3, ADT -4, GDT -5, IDT -6.5, JDT -10,
+                NZDT -13
+            solar_time: A boolean to use local solar time
```

Comment:
I would add more information here from the docs: If set to True then the time is preceded by a plus sign (’+’), then it is interpreted as local solar time instead.